src/main/resources/assets/croptopia/lang/en_nz.json

```diff
@@ -0,0 +1,632 @@
+  "advancements.croptopia.food_press.title": "Blendern't",
+  "advancements.croptopia.food_press.description": "Make an All Purpose Food Press",
+  "advancements.croptopia.mortar_and_pestle.title": "A Deadly Weapon For... Crushing",
+  "advancements.croptopia.mortar_and_pestle.description": "Make a Mortar and Pestle",
+  "advancements.croptopia.eatbig.title": "Fully Stuffed",
+  "advancements.croptopia.eatbig.description": "Eat anything with five or more ingredients",
+  "advancements.croptopia.eatcrafted.title": "Hearty Meal",
+  "advancements.croptopia.eatcrafted.description": "Eat anything crafted",
+  "advancements.croptopia.gather_desert.title": "Undried Seeds",
+  "advancements.croptopia.gather_desert.description": "Gather all the seeds from the desert",
+  "advancements.croptopia.gather_savanna.title": "Hot Seeds",
+  "advancements.croptopia.gather_savanna.description": "Gather all the seeds from the savanna",
+  "advancements.croptopia.gather_forest.title": "Saplings Lite",
+  "advancements.croptopia.gather_forest.description": "Gather all the seeds from the forest",
+  "advancements.croptopia.gather_jungle.title": "Wild Seeds",
+  "advancements.croptopia.gather_jungle.description": "Gather all the seeds from the jungle",
+  "advancements.croptopia.gather_plains.title": "Really Plain Seeds",
+  "advancements.croptopia.gather_plains.description": "Gather all the seeds from the plains",
+  "advancements.croptopia.gather_swamp.title": "Watery seeds",
+  "advancements.croptopia.gather_swamp.description": "Gather all the seeds from the swamp",
+  "advancements.croptopia.gather_all.title": "Land Seed Record",
+  "advancements.croptopia.gather_all.description": "Gather every seed from Croptopia",
+  "advancements.croptopia.getdrinks.title": "Fancy Water",
+  "advancements.croptopia.getdrinks.description": "Craft any kind of drink",
+  "advancements.croptopia.gather_drinks.title": "Mundane Cocktail",
+  "advancements.croptopia.gather_drinks.description": "Drink every drink",
+  "advancements.croptopia.gather_food.title": "Food Critic",
+  "advancements.croptopia.gather_food.description": "Eat everything there is to eat",
+  "advancements.croptopia.knife.title": "Iron Torch",
+  "advancements.croptopia.knife.description": "Make a Knife",
+  "advancements.croptopia.tofu.title": "Meaty beans",
+  "advancements.croptopia.tofu.description": "Make Tofu",
+  "advancements.croptopia.cinnamon.title": "Oddly tasty sticks",
+  "advancements.croptopia.cinnamon.description": "Strip a cinnamon tree from your local jungle",
+  "advancements.croptopia.salt.title": "Sadly, Won't Give More FPS",
+  "advancements.croptopia.salt.description": "Find Salt in the depths of nearby rivers",
+  "advancements.croptopia.gather_tree_all.title": "Streeking Treeathlon",
+  "advancements.croptopia.gather_tree_all.description": "Gather every sapling Croptopia has to offer",
+  "advancements.croptopia.gather_tree_forest.title": "Your Bargain Bin Saplings",
```

Comment:
apparently this word is used here, but I haven't heard it myself. Sounds a bit like an op shop?